pkg/configurator/client.go

```diff
@@ -163,6 +163,7 @@ func (c *Client) configMapListener() {
 				triggerGlobalBroadcast = triggerGlobalBroadcast || (prevConfigMap.TracingAddress != newConfigMap.TracingAddress)
 				triggerGlobalBroadcast = triggerGlobalBroadcast || (prevConfigMap.TracingEndpoint != newConfigMap.TracingEndpoint)
 				triggerGlobalBroadcast = triggerGlobalBroadcast || (prevConfigMap.TracingPort != newConfigMap.TracingPort)
+				triggerGlobalBroadcast = triggerGlobalBroadcast || (prevConfigMap.PrometheusScraping != newConfigMap.PrometheusScraping)
```

Comment:
Can we add a unit test to catch this?

Reply:
let me check if we have one already

Reply:
I would create a test, which iterates over all values and makes sure that changing these results in a pubsub message.
I'd also make it error out when a new key appears and we have no pubsub message for it. (I think the general case is - a config map key/value change should result in a pubsub messag, right?)

Reply:
test incoming as a separate pr